docs/map_state/tags.md

```diff
@@ -0,0 +1,26 @@
+Atlas allows you to visually and programatically add tags to datapoints. Tags can be added collaboratively by anyone
+allowed to edit your project.
+
+
+You can access and operate on your assigned tags by using the `tags` attribute
+of an AtlasMap.
+
+
+```python
+from nomic import AtlasProject
+
+project = AtlasProject(name='My Project')
+
+map = project.maps[0]
+
+map.tags.get_tags()
```

Comment:
Should we note here that tags returns a dictionary where the keys are the tag names the user adds in the frontend and the values are a list of ids? I found this confusing when I was initially using `get_tags`.

Should we also mention that if the use doesn't provide `id_field` in `map_text`, it's very tough to map from tags to your original dataframe? cc @bmschmidt

Reply:
We should make sure all our examples show people uploading ids.

With the new auto assignment it's harder to screw it up... but still not great.